rdbi-core/src/main/java/com/lithium/dbi/rdbi/RDBI.java

```diff
@@ -14,12 +18,20 @@
 @ThreadSafe
 public class RDBI {
 
+    public static final String TRACER_NAME = "rdbi";
+
     private final Pool<Jedis> pool;
     private static final Logger logger = LoggerFactory.getLogger(RDBI.class);
 
     final ProxyFactory proxyFactory;
+    final Tracer tracer;
 
     public RDBI(Pool<Jedis> pool) {
+        this(pool, GlobalOpenTelemetry.get().getTracer(TRACER_NAME));
```

Comment:
If OpenTelemetry is not configured, GlobalOpenTelemetry.get() will return a no-op tracer.